nf_core/components/components.py

```diff
@@ -0,0 +1,179 @@
+import logging
+import os
+import shutil
+from pathlib import Path
+
+import yaml
+
+from nf_core.modules.modules_json import ModulesJson
+from nf_core.modules.modules_repo import ModulesRepo
+
+from .components_utils import get_repo_type
+
+log = logging.getLogger(__name__)
+
+
+class Components:
```

Comment:
Should we name the class `Component` to be consistent with `ModuleCommand` and `SubworkflowCommand`? @mashehu @awgymer

Reply:
I think perhaps yes, maybe even `ComponentCommand` so it's clear what is a base for?

Reply:
Sounds good, I just changed to `Component` but will do this other change :)